openedx/features/edly/validators.py

```diff
@@ -0,0 +1,53 @@
+from logging import getLogger
+
+from django.db import DataError
+from django.db.models import Q
+
+from openedx.features.edly.models import (
+    EdlyUserProfile,
+    EdlySubOrganization
+)
+
+logger = getLogger(__name__)
+
+
+def validate_user_access_with_current_edly_sub_organization(request, possibly_authenticated_user):
```

Comment:
No need to add this method, We have already introduce a method `user_has_edly_organization_access` which validates edly user's access based on its linked edly sub organization (https://github.com/edly-io/edx-platform/pull/73/files)

How ever we will need to add try catch logic for `EdlyUserProfile` since I have verified that new users added from Django admin don't have edly profile.
```python
    if not request.user.edly_profile:
        return False
```

CC: @asadmanzoor93

Reply:
@Zubair Afzal The reason for not using method user_has_edly_organization_access is that, it checks user permission by comparing to the organization set at the time of login (edly user info cookie is set) but in this case, we need to check permission between login process (edly user info cookie is not set).

Reply:
@iam-mhaseeb got it. thanks for the explanation. Let change the name of the method to `is_edly_user_allowed_to_login`